airflow/config_templates/default_celery.py

```diff
@@ -75,6 +85,6 @@ def _broker_supports_visibility_timeout(url):
                            'SSL and/or have all necessary certs and key ({}).'.format(e))
 
 result_backend = DEFAULT_CELERY_CONFIG['result_backend']
-if 'amqp' in result_backend or 'redis' in result_backend or 'rpc' in result_backend:
+if 'amqp://' in result_backend or 'redis://' in result_backend or 'rpc://' in result_backend:
```

Comment:
This change is to address cases like `mysql://amqp:password@localhost` (i.e., "amqp" or "redis" may happen to be part of my user name).

It's not really relating to this PR subject. But this may not be worthwhile to open another PR.